Nos/Models/FlagOption.swift

```diff
@@ -67,19 +63,86 @@ struct FlagOption: Identifiable, Equatable {
         )
     ]
 
+    /// `FlagOption` instances representing different categories of the visibility of a flagged account.
+    static let flagAccountVisibilityCategories: [FlagOption] = [
+        FlagOption(
+            title: String(localized: .localizable.flagAccountMuteTitle),
+            description: String(localized: .localizable.flagAccountMuteDescription),
+            info: nil,
+            category: .visibility(.mute)
+        ),
+        FlagOption(
+            title: String(localized: .localizable.flagAccountDontMuteTitle),
+            description: String(localized: .localizable.flagAccountDontMuteDescription),
+            info: nil,
+            category: .visibility(.unmute)
+        )
+    ]
+
     static func == (lhs: FlagOption, rhs: FlagOption) -> Bool {
         lhs.id == rhs.id
     }
 }
 
+extension FlagOption {
+    static let spamCategory = FlagOption(
```

Comment:
I wonder if we need the word "Category" at the end of these names. I think they might be OK without.